CHANGELOG.txt

```diff
@@ -82,6 +82,11 @@
                 - Form submit error responses with HTML can either replace the default overlay content or the frame content.
                 - New creme.dialog.Frame option 'fillOnError'; if enabled the html error response replaces the content.
                 - New creme.dialog.Dialog option 'fillFrameOnError'; if enabled the html error response replaces the content.
+            * Dialog:
+                - Add option 'propagateEvent' to send creme.dialog.Dialog events to the DOM (the dialog element if opened & window.document).
+                - Add option 'id' to identify the dialog DOM element.
+                - Add access to fitToFrameSize, resize methods from 'ui.dialog' widget.
+                - Add 'cremeInstance' method to 'ui.dialog' widget that returns the creme.dialog.Dialog instance
```

Comment:
- Keep these Changelog lines for the backports (very small features + 2.7 will be released in 8 months => it's 2.6 features).
- Modify the changelog in the related commits, not in a separated commit.